about.html

```diff
@@ -34,11 +31,16 @@
 		</p>
 		<div class="row photos">
 			{% for member in site.team %}
-				<a class="col-xs-6 col-sm-3 col-md-2" href="{{ site.baseurl }}/team/{{ member.short-name }}.html">
-					<img src="{{ site.baseurl }}/img/team/{{ member.short-name }}.jpg">
+				<a class="col-xs-6 col-sm-3 col-md-2" href="{{ site.baseurl }}/team/{{ member.name }}.html">
```

Comment:
You should be able to use ``{{ member.url }}``, similarly for any other collection.